sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -343,7 +343,7 @@ export class SessionReceiver {
   }
 
   /**
-   * Renews the lock on the session to be set to 30 seconds from the moment of invocation.
+   * Renews the lock on the session to extend by the lock duration value as set on Service Bus entity.
```

Comment:
To keep things consistent, can we use the similar wording as in the docs for `renewMessageLock`?

Reply:
done